Mathlib/RingTheory/TensorProduct/MvPolynomial.lean

```diff
@@ -187,7 +190,7 @@ noncomputable def scalarRTensorAlgEquiv :
   rTensorAlgEquiv.trans (mapAlgEquiv σ (Algebra.TensorProduct.lid R N))
 
 variable (R)
-variable (A : Type*) [CommSemiring A] [Algebra R A]
+variable {σ : Type*} (A : Type*) [CommSemiring A] [Algebra R A]
```

Comment:
There is already a sigma around. Declaring another one is not a good idea.

Reply:
Fixed by adding a section above.